about/policy/data-sharing-agreement/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: State Geographic Information Database (SGID) Data Sharing Agreement
+layout: page_nocrumb
+author:
+  display_name: UGRC Office
+  email: ugrc@utah.gov
+tags:
+  - license
+  - policy
+categories: []
+date: 2021-07-16 17:00:00 -0700
+---
+
+## Applicability
+{: .text-left}
+
+The following Data Sharing Agreement applies to all individuals and organizations sharing data with the SGID in any way.
+
+In order to reduce the friction of adding data to the SGID, by submitting a [Porter issue](https://github.com/agrc/porter/issues) to introduce your data to the SGID, you consent to this Data Sharing Agreement. In cases where UGRC submits the Porter issue on your behalf, UGRC will contact you via email to get your consent to this agreement.
```

Comment:
this sentence seems too complex to me.  i like what it's saying, but it took me a few reads to get it.  

I wonder if it would read the same without the intro "In order to reduce the friction of adding data to the SGID"?

I think the main idea is that you are consenting to this data-sharing agreement when sharing data into the sgid.

maybe the next paragraph goes into how to share... you submit the porter form or we can do it for you.

?? just a thought.  but, honestly, it looks great the way you have it.  I'm just nit-picking.

Reply:
No, I think you're right, it is complex. I'll take one last stab at making it more clear.